Sources/NIOHTTP2/HTTP2PipelineHelpers.swift

```diff
@@ -77,6 +78,14 @@ public extension ChannelPipeline {
     }
 }
 
+private final class ErrorHandler: ChannelInboundHandler {
```

Comment:
I think we shouldn't provide one of these ourselves: we should just make sure the user is able to put one where they need to.